packages/hardhat-plugin/src/index.ts

```diff
@@ -221,15 +221,15 @@ ignitionScope
 
 ignitionScope
   .task("status")
-  .addParam("id", "The id of the deployment to show")
+  .addParam("deploymentId", "The id of the deployment to show")
```

Comment:
Unrelated, but this one could be positional, right?

Reply:
Switched to positional